### PR TITLE
Add GUI option comparison to GUI landing page

### PR DESCRIPTION
**Pull request by brian-hay**
See: https://forum.babylonjs.com/t/introducing-the-gui-editor-alpha/24578/82?u=inteja

### CODE REVIEW

**Comment by sebavan** [comment]
Adding @PirateJC and @msDestiny14 to review

File: content/How_To/scene/Multi_Canvases.md [hunk 1]
```diff
@@ -68,6 +68,16 @@ engine.inputElement = myNewElement;
 scene.attachControl();
 ```
 
+## Performance
+
+To ensure optimal performance, **multiple canvases on the same page should have the same size**, otherwise it forces a resize of the master canvas for each view, which is detrimental to performance.
+
+From Babylon.js v5 views can also be disabled, for instance to improve performance when one or more canvases are offscreen.
+
+```
```
Comment (RaananW) [inline content/How_To/scene/Multi_Canvases.md]:
```suggestion
```javascript
```

File: content/landing_pages/divingDeeperGUILandingPage.md [hunk 2]
```diff
@@ -39,8 +39,63 @@ The external [dat.GUI](https://github.com/dataarts/dat.gui) is integrated within
...
+* Looser integration with 3D scene
+* Can’t have GUI elements directly within 3D scene (e.g. applied to meshes)
+* Can’t apply 3D post processing effects to overlaid HTML GUI elements
+* Can’t be used for fullscreen/native VR
+
+### Babylon 2D GUI
+
+#### Pros
+
+* There’s now an awesome [GUI Editor](/toolsAndResources/tools/guiEditor) to make interface creation easier!
```
Comment (RaananW) [inline content/landing_pages/divingDeeperGUILandingPage.md:78]: truly awesome :-)

**Review by RaananW — APPROVED** [review]

**Comment by RaananW** [comment]
@msDestiny14 - I'll wait for your review before merging this.

**Comment by msDestiny14** [comment]
I had a small nit but overall love this! I approve content wise! 

Only thing I'm not 100% sure on is grammar because I'm not an expert at it but it LGTM.

**Comment by brian-hay** [comment]
@msDestiny14 that's valid. Happy to do another grammar pass. I admittedly just copied and pasted much of this from my forum post which was in more of a rough notes form.

**Pull request merged**